pkgs/applications/networking/listadmin/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib, stdenv, fetchurl, makeWrapper, perlPackages }:
+
+stdenv.mkDerivation rec {
+  pname = "listadmin";
+  version = "2.73";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/listadmin/${version}/listadmin-${version}.tar.gz";
+    sha256 = "00333d65ygdbm1hqr4yp2j8vh1cgh3hyfm7iy9y1alf0p0f6aqac";
+  };
+
+  buildInputs = [ perlPackages.perl ];
+  nativeBuildInputs = [ makeWrapper ];
+
+  configurePhase = ''
+  '';
+
+  buildPhase = ''
+    true
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin $out/share/man/man1
+    install -m 755 listadmin.pl $out/bin/listadmin
+    gzip -c listadmin.1 > $out/share/man/man1/listadmin.1.gz
```

Comment:
I don't think you need this. The `compress-man-pages.sh` hook should do that for you in the `fixupPhase`

Reply:
Thanks. I guess I should also fix https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/sieve-connect/default.nix#L30 (in a separate PR), right?

Reply:
Oof good point

Reply:
PR in https://github.com/NixOS/nixpkgs/pull/133605